nilearn/reporting/tests/test_html_report.py

```diff
@@ -11,6 +11,7 @@
     generate_maps,
     generate_random_img,
 )
+from nilearn.experimental.surface import SurfaceLabelsMasker, SurfaceMasker
```

Comment:
Will this import still be from `nilearn.experimental` or those will also be moved to main API?

Reply:
the whole experimental will be moved but I am currently moving things one by one to make it easier to review (while also trying to avoid merge conflicts)